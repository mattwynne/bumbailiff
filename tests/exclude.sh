#!/bin/bash
source "functions.sh"
set_git_date "$(yesterday)"
create_git_repo
echo """
// TODO: fix elixir
""" >file.ex
git add .
git commit --quiet --allow-empty -m "Add file.ex"
echo """
// TODO: fix javascript
""" >file.js
git add .
git commit --quiet --allow-empty -m "Add file.js"
echo """
# TODO: fix ruby
""" >file.rb
git add .
git commit --quiet --allow-empty -m "Add file.rb"
BUMBAILIFF_EXTENSIONS="*.ex *.rb" bumbailiff
echo $?
