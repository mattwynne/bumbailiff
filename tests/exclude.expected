[1mfile.ex:2[0m
[0;32m1 days old[0;37m <a-commit-ref> (Daisy Developer)[0m
TODO: fix elixir

[1mfile.rb:2[0m
[0;32m1 days old[0;37m <a-commit-ref> (Daisy Developer)[0m
TODO: fix ruby

ðŸŒ± [0;32m2 TODO-days accumulated (max allowance: 14 ðŸ’µ).[0m
0
